test/system/errorService/service.js

```diff
@@ -14,12 +14,11 @@
 
 'use strict'
 
-var mu = require('../../../core/core')()
-var mue = require('mu-error')()
+var mu = require('../../../packages/mu')()
```

Comment:
if we wanted to just `require('mu')` we'd have to add the global tests to packages/mu-tests or something, and make that private (so it's no pub'd to npm) 

but that's overloading the `packages` semtantic